packages/mui-material-next/src/styles/typescale.ts

```diff
@@ -0,0 +1,77 @@
+const mdSysTypescale = {
+  label: {
+    small: {
+      family: 'Roboto',
+      weight: '500',
+    },
+    medium: {
+      family: 'Roboto',
+      weight: '500',
+    },
+    large: {
+      family: 'Roboto',
+      weight: '500',
+      lineHeight: 20,
+      size: 14,
+      tracking: 0.1,
+    },
+  },
```

Comment:
```suggestion
  label: {
    small: {
      family: 'Roboto',
      weight: '500',
      tracking: 0.5,
    },
    medium: {
      family: 'Roboto',
      weight: '500',
      tracking: 0.5,
    },
    large: {
      family: 'Roboto',
      weight: '500',
      lineHeight: 20,
      size: 14,
      tracking: 0.1,
    },
  },
```

Reply:
Nice, I am adding these. I haven't done it yet, as I didn't need them for the button. Thanks!